controllers/disruption_controller.go

```diff
@@ -210,9 +212,15 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		// the injection is being created or modified, apply needed actions
 		controllerutil.AddFinalizer(instance, chaostypes.DisruptionFinalizer)
 		if err := r.Update(context.Background(), instance); err != nil {
+			if isModifiedError(err) {
```

Comment:
Does this part of the code is coverage by unit test?

Reply:
No, it is not